### PR TITLE
Add icon to readme and favicon to generated code

### LINKED ISSUE #28: Create a logo
Having a logo for this project would be great for adding it in app stores like Citadel's or Umbrel's.

### PR DESCRIPTION
**Pull request by jerolimov**
Fixes #28 :)

After merging the readme looks like this:

Screenshot 2021-11-21 at 22 55 18

### CODE REVIEW

**Comment by faxemaxee** [comment]
nice!

File: files/root/index.html [hunk 1]
```diff
@@ -111,6 +111,7 @@
         cursor: pointer;
       }
     </style>
+    <link rel="icon" type="image/svg+xml" href="/lnme/favicon.svg" />
```
Comment (bumi) [inline files/root/index.html:114]: 👍 we should use that also as `og:image` further up in line 9

**Review by bumi — CHANGES_REQUESTED** [review]
yay, thanks for the PR.

File: files/root/index.html (hunk 1 above)
Comment (jerolimov) [reply files/root/index.html:114]: Updated, removed the zap.svg and rebuild the generated code with rice

Comment (bumi) [reply files/root/index.html:114]: should we use the full lnme-icon as `og:image` ? 

in Alby for example: 
image
vs.
image

**Review by bumi — APPROVED** [review]
love it! thanks so much!

**Pull request merged**